### ISSUE TITLE
Timers cannot be stopped from another thread

### ISSUE THREAD

**Issue by TheRustlessSummer** [colmap/colmap#1017]
Hello , when I run stereo in colmap, this question will happen.
`Timers cannot be stopped from another thread`
ubuntu:18.04 ; cuda:10.2 ; nvidia driver : 440
But when I run in the same computer in ubuntu 16.04 , there wasn't this question.
How can I solve this question?

**Comment by TheRustlessSummer** [comment]
The information of my GPU and system：
-ubuntu 18.04
-CUDA 10.2
-GeForce GTX 1660 6GB
-NVIDIA driver 440

First I reduced the number of images from 200 to 30 to 10. But the question still happened.
Then I monitored GPU usage , the memory hasn't been out. The memory only use about 800M. 
So I want to know how can I solve this question.
The question is like below:

`QObject::~QObject: Timers cannot be stopped from another thread`

or sometime was 

`QObject::~QObject: Timers cannot be stopped from another thread`
`*** Aborted at 1603071261 (unix time) try "date -d @1603071261" if you are using GNU date ***`
